src/Xamarin.Android.Build.Tasks/Microsoft.NET.Sdk.Android/WorkloadManifest.in.json

```diff
@@ -14,7 +14,12 @@
         "Microsoft.Android.Templates"
       ],
       "platforms": [ "win-x64", "win-arm64", "linux-x64", "osx-x64", "osx-arm64" ],
-      "extends" : [ "microsoft-net-runtime-android", "microsoft-net-runtime-android-aot" ]
+      "extends" : [ 
+        "microsoft-net-runtime-android-net6",
+        "microsoft-net-runtime-android-aot-net6",
+        "microsoft-net-runtime-android",
+        "microsoft-net-runtime-android-aot"
+      ]
```

Comment:
@rolfbjarne you'll want to take similar changes here on the xamarin-macios side.

Reply:
@jonathanpeppers  I ended up doing something slightly different, I split it in two different workloads: https://gist.github.com/rolfbjarne/4c9f93d892837a9b13e9cd577f8554a7#file-workloadmanifest-json-L3-L37 - although I'm not sure if that will cause any problems? Should I out everything in the `ios` workload instead?

Reply:
You might want to keep them all together for now. If you open a `net6.0-ios` project without a workload installed, this logic will prompt VS to install MAUI:

https://github.com/dotnet/sdk/blob/09b31215867d1ffe4955fd5b7cd91eb552d3632c/src/Tasks/Microsoft.NET.Build.Tasks/ShowMissingWorkloads.cs#L24-L26

I don't think this code would work if there is an `ios-net6` workload.

We might get them to rework this logic in .NET 7 RC 2, so that we can make the .NET 6 packs optional?

Reply:
OK, I'll look into changing that tomorrow.

Reply:
I've changed this in our [cross/multi targeting PR](https://github.com/xamarin/xamarin-macios/pull/15761) for RC 1.